### DISCUSSION TITLE
Valvole termostatiche "OPEN SOURCE" o proprietarie?

### DISCUSSION THREAD

**Discussion by maurominella** [indomus/forum#1819] [HUB: Home Assistant]
So bene che non si deve usare questo forum per pubblicità e recensioni, mi limiterò quindi a raccontare la mia esperienza personale, e se qualcuno vuole menzionare la propria citando il prodotto che ha usato... credo rientri negli obiettivi di questa community ;-).

Insomma io sto trovando davvero seccante il fatto che queste valvole termostatiche zigbee continuino a rompersi.
Sì, le ho acquistate in Cina perché mi sembrano davvero identiche a quelle che posso acquistare su Amazon al doppio di quella cifra (ma consegna quasi in giornata...), e che secondo me provengono sempre da lì: quindi non avendo particolare fretta ne ho prese di diversi modelli per la mia seconda casa, mentre nella casa in città avevo acquistato (prima di scoprire Home Assistant) delle costose MAX! tedesche. Nulla da dire sulla qualità di quel prodotto: solide, mai un problema, si connettorono all'unico hub anche molto distante (sto in una villetta su 4 livelli e non ho mai avuto problemi di connessione), hanno il "loro" termometro da parete automaticamente configurato che evita che la valvola rilevi la temperatura vicino al calorifero, il software (proprietario) è perfetto, c'è pure il servizio di integrazione per Home assistant.
Come però spesso succede con tecnologie proprietarie, le MAX! sono state discontinuate dopo 3 anni che le avevo acquistate, e la EQ3 AG che le produce(va) mi ha fatto una proposta di acquisto scontato per la nuova soluzione che si chiama homematic. Considerato che funzionavano perfettamente e che le avevo pagate circa 100 euro a valvola (e ne ho 11...) più 60 euro di termometro (a stanza), ho evitato di "approfittare" questa "offerta". Però, passato un altro anno, si è rotta una di queste (e ci sta) quindi ora che faccio? Mi compero tutta la nuova serie per 2mila euro (e il rischio di ritrovami in questa situazione fra 3/4 anni). E se le butto tutte, con cosa le sostituisco? L'ideale sarebbe scegliere modelli indipendenti dal software e che si integrano in Home Assistant.

Quindi come dicevo ho provato per la casa al mare a comperare queste cinesi, collegandole ad home assistant. Oggi costano circa 35 euro l'una, un anno fa le trovavo a 20 euro, le stesse identiche, comunque sempre molto meno che in Italia e ripeto secondo me sono le stesse che si trovano nei nostri negozi online. Ho provato questi 3 modelli che vi mostro qui senza link esterni
![image](https://user-images.githubusercontent.com/32428960/183966137-93b1cf37-a428-4c7f-a235-d721d45a18ba.png)
![image](https://user-images.githubusercontent.com/32428960/183966264-c703d7c2-ad0d-4911-b74a-e7d0f6b8df33.png)
![image](https://user-images.githubusercontent.com/32428960/183966342-dc2f104c-69e3-48cf-b502-3012863aecf6.png)

, tanto si possono "mischiare" in Home Assistant. E come dicevo, una dopo l'altra si sono rotte meccanicamente all'interno. D'altra parte sono soggette ad un grande sforzo nell'aprire/chiudere la valvola, e ad alte temperature, e sono dei pezzi di plastica certamente mediocre. Magari sono solo stato sfortunato, ma se ne sono rotte 8 su 16 in 18 mesi. E' innegabile che, almeno queste, non sono minimamente confrontabili con le tedesche che ancora mi funzionano in città dopo 4 anni (a parte una, che ci starebbe benissimo, se potessi acquistarne un'altra).

Quindi ora sono ad un punto fermo: è vero che costano molto meno delle tedesche e che le gestisco come voglio io, però arrendersi vorrebbe dire di nuovo legarsi mani e piedi alle decisioni di aziende che ti "costringono" a prendere le decisioni che vogliono loro, ad usare il loro software, acquistare servizi aggiuntivi a pagamento, a comunicare a loro i tuoi dati.

Insomma mi piacerebbe sentire se qualcuno può menzionare la propria esperienza con valvole termostatiche "OPEN SOURCE" come le zigbee che si trovano nei nostri negozi e in quelli cinesi, da gestire gestisce con Home Assistant non necessariamente super economiche ma che, nel suo caso (magari partendo da una recensione che io non ho trovato), hanno dimostrato di essere di buona qualità e resistere a lungo nel tempo (diciamo 3 anni).

**Comment by snakuzzo** [comment]
Ciao.
Innanzi tutto, "OPEN SOURCE" non vuol dire quello che intendi tu. Ti consiglio di andare a guardare bene cosa si intende con quel termine.

Quanto alle valvole, "il risparmio non è mai guadagno".
Io posso consigliarti le Tado° da prendere in sconto su Amazon (ci vanno spesso). Quelle puoi integrarle con integrazione "Tado°" su HA (che sfrutta il cloud) oppure puoi scegliere di integrarle localmente con [Homkit Controller](https://www.home-assistant.io/integrations/homekit_controller/) (da non confondere con l'integrazione [Homekit](https://www.home-assistant.io/integrations/homekit/)).
Le ho da più di 3 anni e non ho mai avuto alcun problema (come immagino la maggior parte delle persone che le possiede).
Non sei assolutamente obbligato a sottoscrivere abbonamenti e la frase "ti costringono a prendere le decisioni che vogliono loro", permettimi di dire che non ha alcun senso.

**Reply by maurominella** [reply]
Ciao Diego, lavoro come software developer e cloud architect da oltre 20 anni in una delle maggiori software house worldwide, quindi è proprio perché conosco molto bene i principi e le sfumature dell'open source che l'ho associato -naturalmente virgolettato, forse non lo hai notato- a quelle situazioni in cui il prodotto (software o hardware) viene fornito "aperto" ovvero trasparente, personalizzabile e integrabile con sistemi che scegli TU, e non chi te lo ha fornito. Nell'esempio della linea MAX! che menzionavo, invece, non esiste la scelta di agganciare al loro hub di ottima qualità valvole "compatibili" perché chi lo ha sviluppato lo ha esplicitamente impedito per "costringerti" ad acquistare esclusivamente i suoi accessori. Fin qui, si può anche capire questa politica commerciale un po' aggressiva ma tutto sommato anche dichiarata esplicitamente. Ma poi sono andati oltre: di colpo hanno chiuso il supporto e la produzione, sostituendola con una nuova linea non compatibile con la LORO versione precedente in vendita fino al giorno prima, di fatto "costringendomi" (nel senso che non c'è alcuna alternativa), come dicevo, a buttare via un investimento consistente che ancora funziona. 

Quindi ho apprezzato molto queste valvole zigbee provenienti da molti fornitori diversi, che si possono associare ad uno dei tanti hub in circolazione su cui ho patchato in due minuti un [firmware open source](https://tasmota.github.io/docs/) (questa volta nel senso puro del termine) e che si possono utilizzare dall'interno di [un software (anch'esso "VERO" software open source)](https://github.com/home-assistant/core) che gira su una virtual machine nel cloud, ma mi sono scontrato con queste valvole di bassa qualità, dove il produttore spesso è ignoto, senza un indirizzo per il supporto, senza nemmeno un sito WEB o qualcuno cui rivolgersi che si prenda l'impegno di curare un problema all'inizio alla fine, lasciando il "cliente" in balia della buona volontà della community, dove chiunque può dire la sua senza impegno e senza responsabilità. E anche qui, il paragone con l'open source è del tutto appropriato.

Ora, se per il passato sono costretto a fare "quello che vogliono loro" (cioè buttare via i componenti MAX!, visto che via via che si rompono non si possono sostituire e il software di gestione non va più), per il futuro ovviamente la scelta è tutta mia, da cui la mia domanda aperta a questa community per trovare delle valvole zigbee affidabili, che avrei affiancato a quelle già acquistate via via che si stanno rompendo. Grazie quindi per l'indicazione del TADO, ho dato un'occhiata e non mi pare siano integrabili le zigbee, ma il fatto che siano affidabili riportato direttamente da un esperto di domotica (quale io non sono) è un'informazione certamente utile.
Ho provato anche a fare un calcolo di massima per capire quanto costerebbe controllare i miei 11+5 radiatori, inclusi gli accessori come i due controller, i sensori di temperatura delle stanze (che li vorrei separati dalle valvole, come i vecchi MAX!), e temo si superi abbondantemente la cifra della soluzione EQ3 che, mi secca dirlo, sono certo sia pure ottima. Ma come hai detto tu, "chi più spende meno spende" (anche qui virgolettato ;-) )...

**Reply by 50494554524F** [reply]
i termostati non ti servono, puoi usare un termometro qualsiasi... ad esempio, xiaomi termometer 2, piccolo, lo puoi attaccare al muro con calamita volendo perché è leggero, pure qua soffre l'inflazione da 3€ a 5€ circa... quelle valvole a 35€ ormai non convengono più visto che sono senza garanzia, ti conviene aspettare qualche offerta su amazon... compare d'estate le cose per l'inverno e in inverno le cose per l'estate

edit: sfortuna, tado sono già andate in promo mese scorso a 45€...

**Comment by KC-inDomus** [comment]
> So bene che non si deve usare questo forum per pubblicità e recensioni, mi limiterò quindi a raccontare la mia esperienza personale, e se qualcuno vuole menzionare la propria citando il prodotto che ha usato... credo rientri negli obiettivi di questa community ;-).
> 
> Insomma io sto trovando davvero seccante il fatto che queste valvole termostatiche zigbee continuino a rompersi. Sì, le ho acquistate in Cina perché mi sembrano davvero identiche a quelle che posso acquistare su Amazon al doppio di quella cifra (ma consegna quasi in giornata...), e che secondo me provengono sempre da lì: quindi non avendo particolare fretta ne ho prese di diversi modelli per la mia seconda casa, mentre nella casa in città avevo acquistato (prima di scoprire Home Assistant) delle costose MAX! tedesche. Nulla da dire sulla qualità di quel prodotto: solide, mai un problema, si connettorono all'unico hub anche molto distante (sto in una villetta su 4 livelli e non ho mai avuto problemi di connessione), hanno il "loro" termometro da parete automaticamente configurato che evita che la valvola rilevi la temperatura vicino al calorifero, il software (proprietario) è perfetto, c'è pure il servizio di integrazione per Home assistant. Come però spesso succede con tecnologie proprietarie, le MAX! sono state discontinuate dopo 3 anni che le avevo acquistate, e la EQ3 AG che le produce(va) mi ha fatto una proposta di acquisto scontato per la nuova soluzione che si chiama homematic. Considerato che funzionavano perfettamente e che le avevo pagate circa 100 euro a valvola (e ne ho 11...) più 60 euro di termometro (a stanza), ho evitato di "approfittare" questa "offerta". Però, passato un altro anno, si è rotta una di queste (e ci sta) quindi ora che faccio? Mi compero tutta la nuova serie per 2mila euro (e il rischio di ritrovami in questa situazione fra 3/4 anni). E se le butto tutte, con cosa le sostituisco? L'ideale sarebbe scegliere modelli indipendenti dal software e che si integrano in Home Assistant.
> 
> Quindi come dicevo ho provato per la casa al mare a comperare queste cinesi, collegandole ad home assistant. Oggi costano circa 35 euro l'una, un anno fa le trovavo a 20 euro, le stesse identiche, comunque sempre molto meno che in Italia e ripeto secondo me sono le stesse che si trovano nei nostri negozi online. Ho provato questi 3 modelli che vi mostro qui senza link esterni ![image](https://user-images.githubusercontent.com/32428960/183966137-93b1cf37-a428-4c7f-a235-d721d45a18ba.png) ![image](https://user-images.githubusercontent.com/32428960/183966264-c703d7c2-ad0d-4911-b74a-e7d0f6b8df33.png) ![image](https://user-images.githubusercontent.com/32428960/183966342-dc2f104c-69e3-48cf-b502-3012863aecf6.png)
> 
> , tanto si possono "mischiare" in Home Assistant. E come dicevo, una dopo l'altra si sono rotte meccanicamente all'interno. D'altra parte sono soggette ad un grande sforzo nell'aprire/chiudere la valvola, e ad alte temperature, e sono dei pezzi di plastica certamente mediocre. Magari sono solo stato sfortunato, ma se ne sono rotte 8 su 16 in 18 mesi. E' innegabile che, almeno queste, non sono minimamente confrontabili con le tedesche che ancora mi funzionano in città dopo 4 anni (a parte una, che ci starebbe benissimo, se potessi acquistarne un'altra).
> 
> Quindi ora sono ad un punto fermo: è vero che costano molto meno delle tedesche e che le gestisco come voglio io, però arrendersi vorrebbe dire di nuovo legarsi mani e piedi alle decisioni di aziende che ti "costringono" a prendere le decisioni che vogliono loro, ad usare il loro software, acquistare servizi aggiuntivi a pagamento, a comunicare a loro i tuoi dati.
> 
> Insomma mi piacerebbe sentire se qualcuno può menzionare la propria esperienza con valvole termostatiche "OPEN SOURCE" come le zigbee che si trovano nei nostri negozi e in quelli cinesi, da gestire gestisce con Home Assistant non necessariamente super economiche ma che, nel suo caso (magari partendo da una recensione che io non ho trovato), hanno dimostrato di essere di buona qualità e resistere a lungo nel tempo (diciamo 3 anni).

a qualità si paga. Io uso [Tado°](https://indomus.it/componenti/testa-termostatica-intelligente-tado/) da 4 anni e rimane tra gli investimenti migliori che abbia fatto, sin qui, in domotica.

**Reply by maurominella** [reply]
Ottimo, era quello che volevo sapere. Guardando questa immagine però mi resta qualche dubbio
![image](https://user-images.githubusercontent.com/32428960/184100834-0550da5d-f43d-4c54-81ef-c870dea4c5c1.png)

1. nella stanza con 22 gradi è presente 1 testa collegata al calorifero più 1 sensore di temperatura: questo significa che i due oggetti sono "legati" e quindi la valvola "legge" la temperatura della stanza dal sensore sulla parete, corretto? Ed ho inteso correttamente il fatto che la temperatura si può impostare sempre dal sensore di temperatura e non solo dalla testa del termosifone?
2. nella stanza con 18, 20 e 21 gradi, c'è solo la testa sul termostato. Quindi in questo caso la temperatura viene rilevata solo nelle vicinanze del calorifero, corretto? Hai avuto evidenza di temperatura rilevata male per il fatto che il sensore è attaccato al calorifero?
3. Immagino che il bridge sia unico per tutta la casa per trasmettere/ricevere informazioni nel cloud. Io ho una casetta su 4 livelli, dalla taverna alla mansarda, metterei il bridge a metà strada. In caso non fosse sufficiente, si può pensare di metterne due oppure mi troverei poi ad avere due "case" separate nella app?
4. Qual è la funzione dell'apparecchio vicino alla caldaia, spegnere la caldaia quando tutte le teste sono chiuse? Nel mio caso la caldaia ha già un sistema di protezione che la "addormenta" senza danni se tutti i caloriferi sono chiusi e lei è accesa, quindi mi chiedevo se posso farne a meno oppure se è necessario per poter usare questo sistema.

**Reply by snakuzzo** [reply]
Quello che vedi di forma quadrata non è un sensore, ma il termostato.
La valvola legge la temperatura del "suo" sensore...che potrebbe differire da quella di un eventuale sensore esterno.
Per ovviare a questo problema, puoi sfruttare le API (in cloud) per fare l'override della temperatura in maniera dinamica tramite automazione. Per farlo devi fare qualche request in POST per recuperare il token e poi puoi settarla come ti pare (già fatto per una cosa simile).

Per quanto riguarda il bridge...in pratica tutte le valvole si connettono a quel bridge. Serve sia per il cloud che per un eventuale configurazione con homekit controller (come ti dicevo nella mia prima risposta).
Il bridge non è né Wi-Fi, né zigbee. Viene collegato via LAN e poi comunica con le teste via frequenze radio da 868 MHz (leggi [QUI](https://support.tado.com/it/articles/3557440-come-comunica-l-internet-bridge-tado-con-altri-dispositivi-tado)).
Quanto al discorso sulla eventuale mancata copertura, devi considerare che ogni bridge è una "Home Tado". Quindi sarebbero due gestioni separate (per un approfondimento leggi [QUI](https://support.tado.com/it/articles/3455030-quanti-dispositivi-posso-avere-nella-mia-home-tado))

**Reply by maurominella** [reply]
@snakuzzo tu dici "quello di forma quadrata", però se vedi bene nella foto ci sono due sensori quadrati, e se ho inteso bene sono completamente diversi: il primo è quello che dici tu ovvero il termostato intelligente, che si collega alla caldaia via cavo, e dall'altra si collega senza fili alle teste sui termosifoni. Serve a spegnere la caldaia quando TUTTE le teste dei caloriferi sono chiuse, e ad accenderla quando almeno una testa si apre, e serve a inviare la temperatura desiderata all teste secondo la programmazione settimanale impostata con la app, perché non credo che le teste siano autonome oppure vengano comandate dal cloud o dalla app ogni volta che devono cambiare la temperatura. Per la cronaca, così è esattamente anche come funziona(va) il sistema MAX! che citavo all'inizio.

Il secondo scatolino quadrato (esteticamente identico al termostato intelligente) invece è il sensore di temperatura/umidità che si accoppia (senza fili, infatti ha il segno del wifi) ad una o più teste. Serve ad evitare che la testa rilevi la temperatura con il suo sensore interno, e serve anche a comandare la valvola senza toccare la testa sul calorifero ma solo agendo sul sensore a parete, e serve anche nel caso in cui in una stanza ci siano più caloriferi perché li associ tutti allo stesso sensore di temperatura. Anche in questo caso, questo è lo stesso funzionamento delle MAX!

Quindi, sempre se ho inteso bene, il primo scatolino quadrato (termostato intelligente) è INDISPENSABILE per il funzionamento di tutto il sistema, mentre il secondo (rilevatore di temperatura/umidità) è un accessorio OPZIONALE DELLE TESTE, da usare solo se non ti fidi che la testa rilevi la temperatura corretta.

Correggetemi per piacere se ho capito male, ma da questo ragionamento concludo che per poter configurare la mia casa al mare che ha 5 caloriferi, dovrei acquistare:
- 5 teste (obbligatorio)
- 1 termostato intelligente cablato o wireless per la caldaia (obbligatorio)
- 1 bridge (obbligatorio)
- 5 rilevatori di temperatura (opzionale)

Insomma, a meno di sconti e bundle, spenderei circa 1.000 euro, di cui 400 euro solo di rilevatori di temperatura opzionali. Cavoli...

**Reply by snakuzzo** [reply]
io le teste le uso senza termostato, avendo il centralizzato... quindi "sì" le puoi usare anche indipendentemente.
Il termostato volendo puoi realizzarlo a costo molto più basso usando uno shelly e un sensore di temperatura esterno (tipo aqara). E poi su HA generi l'entità di tipo generic thermostat.

Se hai l'autonomo ricordati che in base a come hai l'impianto dovrai comunque tenere una testa con valvola sempre aperta.

Se parli di quello a sinistra (rettangolare), in effetti non l'avevo mai visto prima :)

**Comment by maurominella** [comment]
No, io parlo del secondo da destra indicato qui dalla freccia rossa, che è un [sensore di temperatura e umidità](https://amzn.to/3pcsAAf) da collegare wireless alla testa sul termosifone:
![image](https://user-images.githubusercontent.com/32428960/184173889-543ec483-b77e-4c3c-854e-65be5b717a7c.png)

, da non confondere con il secondo da sinistra che è sempre quadrato, ma è un'altra cosa ovvero un [termostato intelligente](https://indomus.it/componenti/termostato-intelligente-tado/) da collegare via cavo alla caldaia o al riscaldamento a pavimento.

Quello rettangolare invece è un ricevitore collegato con un cavo direttamente alla caldaia, che riceve i segnali wireless per comandarla attraverso un dispositivo simile al secondo da sinistra, ma wifi. Questo posso escluderlo perché la mia caldaia autonoma è già collegata ad un termostato con un contatto pulito.

Hai ragione che potrei spendere meno costruendo uno shelly, ma vorrebbe dire usare HA per controllare continuamente la temperatura, se manca la connessione o si blocca la macchina dopo un update, salta tutto. Il termostato intelligente serve proprio ad evitare tutto questo integrandolo in uno scatolino che non richede particolari attenzioni e posizioni in casa.

Ho fatto i calcoli precisi con i bundle e gli sconti al momento disponibili: per 5 termosifoni, il bridge e il termostato dovrei spendere 625 euro più eventualmente i 5 termostati wifi (che sono altri 402 euro). Davvero tanto. Non per tornare al punto di partenza, ma se solo ci fossero state delle valvole zigbee decenti avrei potuto davvero risparmiare e divertirmi.

**Reply by snakuzzo** [reply]
Ok sì, hai ragione. Certo l'hanno fatto uguale al termostato :)
Comunque...alla fine devi valutare tu ..ognuno ha le proprie esigenze e le proprie possibilità.
Io 5 valvole le ho pagate in totale 200 euro in offerta al prime day su Amazon. Contando anche il bridge, circa 250.
A mio avviso, se hai intenzione di risparmiare qualcosa, potresti anche risparmiare sui sensori esterni e tarare le singole valvole con l'offset utilizzando un sensore da 10 euro per fare qualche misurazione. Magari ti accorgi che la differenza è davvero minima (come nel mio caso...stavo su 0.1/0.2 ºC) e sei a posto.

**Reply by maurominella** [reply]
Davvero una super offerta, quasi il 50% di quanto costano oggi, hai fatto davvero un ottimo affare.

Il problema dell'offset è che servirebbe un offset di spegnimento ma anche uno di accensione. Io ho riscontrato che dopo che la stanza si è scaldata è normale, ad 1cm al termosifone, avere anche 2 o 3 gradi in più rispetto al centro della stanza, non 0.2 e non è colpa della valvola, poverina rileva semplicemente la temperatura che c'è. Ma se imposti un offset a +2 e la temperatura minima a 19, poi ti si spegne appena raggiunge i 17. In salita l'offset quindi non dovrebbe essere usato perché in accensione la temperatura sul termosifone è identica a quella della stanza, è una cosa che hanno chiesto in molti.

Da questa lunga conversazione ho concluso che:
1. basta impegnarsi a cercare valvole zigbee di qualità, costano poco e valgono altrettanto. Vanno bene solo per giocare
2. le Tado sono davvero troppo costose, si possono prendere se trovi un'offerta o se rinunci a qualche servizio (tipo il sensore di temperatura esterno che, si sarà capito, per me è importante). Ma non posso spendere 86,70 + 80, 50 = 167,20 euro per ogni valvola, più l'hub + il bridge (220 euro con il kit base). Con 5 termosifoni nella casa al mare e 12 termosifoni nella villetta devo chiedere un mutuo...
3. tutto sommato le MAX! che mi hanno fatto arrabbiare per i motivi che dicevo sopra, ma che devo riconoscere sono di ottima qualità, hanno un prezzo notevolmente più basso (45% in meno) sia per valvole e termometro esterno (41,29+49,95), sia per l'hub che è unico (61,23, risparmio il 72%) e non doppio (hub + bridge) come nel Tado

Andrò quindi avanti con l'opzione 3. Grazie @snakuzzo  e tutti per i contributi

**Reply by 50494554524F** [reply]
se vuoi le cinesi il 22 comincia il periodo di sconti estivi, puoi dare uno sguardo, già si trovano intorno a 29-30€...
